runtimeconfig/google/cloud/runtimeconfig/config.py

```diff
@@ -0,0 +1,291 @@
+                       to the ``client`` stored on the current config.
+
+        :rtype: :class:`google.cloud.runtimeconfig.variable.Variable` or None
+        :returns: The variable object if it exists, otherwise None.
+        """
+        client = self._require_client(client)
+        variable = Variable(config=self, name=variable_name)
+        try:
+            response = client.connection.api_request(
+                method='GET', path=variable.path, _target_object=variable)
+            # NOTE: We assume response.get('name') matches `variable_name`.
+            variable._set_properties(api_response=response)
+            # NOTE: This will not fail immediately in a batch. However, when
+            #       Batch.finish() is called, the resulting `NotFound` will be
+            #       raised.
+            return variable
+        except NotFound:
+            return None
+
+    def list_variables(self, page_size=None, page_token=None, filter=None,
+                   fields=None, client=None):
+        """API call:  list variables for this config.
+
+        This only lists variable names, not the values.
+
+        See:
+        https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/v1beta1/projects.configs.variables/list
+
+        :type page_size: integer or ``NoneType``
+        :param page_size: maximum number of variables to return.
+
+        :type page_token: string
+        :param page_token: opaque marker for the next "page" of variables. If not
+                           passed, will return the first page of variables.
+
+        :type filter: string or ``NoneType``
+        :param filter: optional filter used to filter variables.
+
+        :type fields: string or ``NoneType``
+        :param fields: Selector specifying which fields to include in a
+                       partial response. Must be a list of fields. For example
+                       to get a partial response with just the next page token
+                       and the name of each variable returned:
+                       'variables/name,nextPageToken'
+
+        :type client: :class:`~google.cloud.runtimeconfig.client.Client` or
+                      ``NoneType``
+        :param client: Optional. The client to use.  If not passed, falls back
+                       to the ``client`` stored on the current config.
+
+        :rtype: tuple, (list, str)
+        :returns: list of
+                  :class:`~google.cloud.runtimeconfig.variable.Variable`, plus
+                  a "next page token" string:  if not ``None``, indicates that
+                  more variables can be retrieved with another call (pass that
+                  value as ``page_token``).
+        """
+        params = {}
+
+        if page_size is not None:
+            params['pageSize'] = page_size
+
+        if page_token is not None:
+            params['pageToken'] = page_token
+
+        if filter is not None:
+            params['filter'] = filter
```

Comment:
Done. Yeah, I know it's a built-in, but didn't realize we didn't want to alias them.